Add armhf Debian package to release

hnw opened the pull request:
Add armhf(32bit ARM with hardfloat) Debian package support.

The package has already been tested in two weeks on my own Raspberry Pi 2, and it works fine.

However, this pull request has one problem. This change requires Docker image `yhnw/docker-mackerel-deb-builder`. We should use Docker image which is managed by Mackerel dev team.

If the dev team approve mackerelio/docker-mackerel-deb-builder#1, then I'll fix the problem.

astj commented:
[memo]
- In Debian "armhf" supports ARMv7~ https://www.debian.org/ports/arm/
  - "armhfp" for CentOS also supports ARMv7~ https://wiki.centos.org/SpecialInterestGroup/AltArch/armhfp
- From those point of view, ARMv7 seems enough, but:
- Raspberry Pi 1 (and zero?) are ARMv6 https://en.wikipedia.org/wiki/Raspberry_Pi#Specifications
- For those ARMv6 family, Raspberry Pi teams maintain Debian fork (Raspbian) for armhf with ARMv6 https://www.raspbian.org/RaspbianFAQ#What_is_Raspbian.3F

astj commented:
@hnw Thank you for pull request!  Adding 32bit arm packages seems good, but maybe we need some discussion around GOARM parameter. Please check the review comment above.

hnw commented:
Hi @astj. Thanks for the comment.

I fixed a Docker image and added comments.

As you mentioned above, ARMv6 is called as armhf on Raspbian. Raspberry Pi is widely used, so I think we should support them.

astj commented:
Sorry to have kept you waiting so long. LGTM!
